substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/JavaLangSubstitutions.java

```diff
@@ -392,88 +392,88 @@ private static void setSecurityManager(SecurityManager s) {
 }
 
 @TargetClass(java.lang.StrictMath.class)
-@CLibrary(value = "strictmath", requireStatic = true)
+@CLibrary(value = "m", requireStatic = false)
```

Comment:
@galderz had to use `"libm"` on Mac, but this resulted in `-llibm` on Linux, while we need `-lm` for it to work.
This will probably be an issue across platforms.